app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -60,7 +60,17 @@ interface Pixel {
         SURVEY_CTA_SHOWN(pixelName = "mus_cs"),
         SURVEY_CTA_DISMISSED(pixelName = "mus_cd"),
         SURVEY_CTA_LAUNCHED_SURVEY(pixelName = "mus_cl"),
-        SURVEY_SURVEY_DISMISSED(pixelName = "mus_sd")
+        SURVEY_SURVEY_DISMISSED(pixelName = "mus_sd"),
+
+        AUTOMATIC_CLEAR_DATA_WHAT_OPTION_NONE("macwhat_n"),
+        AUTOMATIC_CLEAR_DATA_WHAT_OPTION_TABS("macwhat_t"),
+        AUTOMATIC_CLEAR_DATA_WHAT_OPTION_TABS_AND_DATA("macwhat_a"),
```

Comment:
Could we make this `macwhat_d` using `d` for data?

Reply:
That makes sense to me 👍 

(FYI, I was using `a` for `all`. I realise now that probably wasn't obvious)